envs/environment.yaml

```diff
@@ -77,7 +77,7 @@ dependencies:
 
 # Default solver for tests (required for CI)
 - glpk
-- ipopt<3.13.3
+- ipopt
```

Comment:
Dropping the limitation. Please note quickly as a comment here on github: Why is this limitation now lifted?